src/api/app/models/user.rb

```diff
@@ -64,6 +64,7 @@ class User < ApplicationRecord
   has_many :rss_feed_items, -> { order(created_at: :desc) }, class_name: 'Notification::RssFeedItem', as: :subscriber, dependent: :destroy
 
   has_and_belongs_to_many :announcements
+  has_many :commit_activities
```

Comment:
Because that's how the model is called

Reply:
why is the model called like that?

Reply:
because we don't track individual commits - it's arguably, but we found it wrong to name something Commit if it's not about a commit, but about the activity of a day.